controllers/servers/errors.py

```diff
@@ -32,3 +33,11 @@ class ObjectAlreadyProcessingError(BaseControllerServerException):
     def __init__(self, object_id_or_name):
         super().__init__()
         self.message = messages.OBJECT_ALREADY_PROCESSING_MESSAGE.format(object_id_or_name)
+
+
+class RequiredBytesMismatch(BaseArrayActionException):
```

Comment:
shouldn't we return [INVALID_ARGUMENT](https://github.com/container-storage-interface/spec/blob/v1.5.0/spec.md#createvolume-errors) for this error?

Reply:
right, done